website/siteConfig.js

```diff
@@ -270,25 +245,37 @@ const users = [
 
 const siteConfig = {
   title: 'Jest',
+  tagline: '🃏 Delightful JavaScript Testing',
   url: 'https://facebook.github.io',
   baseUrl: '/jest/',
+  projectName: 'jest',
   repo: 'facebook/jest',
-  githubButton,
   users,
+  editUrl: 'https://github.com/facebook/jest/edit/master/docs/',
+  headerLinks: [
+    {doc: 'getting-started', label: 'Docs'},
+    {doc: 'api', label: 'API'},
+    {page: 'help', label: 'Help'},
+    {blog: true, label: 'Blog'},
+    {languages: true},
+    {search: true},
+    {href: 'https://github.com/facebook/jest', label: 'GitHub'},
+  ],
+  headerIcon: 'img/jest.svg',
+  footerIcon: 'img/jest.svg',
+  favicon: 'img/favicon/favicon.ico',
+  ogImage: 'img/opengraph.png',
+  recruitingLink: 'https://crowdin.com/project/jest',
+  algolia: {
+    apiKey: '833906d7486e4059359fa58823c4ef56',
```

Comment:
Should we expose Algolia API key?

Reply:
Good point. This would be the same issue as you not having crowdin keys to test. I'll move the keys to CI.